src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TypeChar.cs

```diff
@@ -357,12 +372,12 @@ private bool IsFilterCharacter(char ch)
             return GetCompletionRules().IsFilterCharacter(selectedItem, ch, filterText);
         }
 
-        private string GetCurrentFilterText(Model model, CompletionItem selectedItem)
+        private string GetCurrentFilterText(Model model, CompletionItem selectedItem, bool stopBeforeCaret = true)
         {
             var textSnapshot = this.TextView.TextSnapshot;
             var viewSpan = model.GetSubjectBufferFilterSpanInViewBuffer(selectedItem.FilterSpan);
             var filterText = model.GetCurrentTextInSnapshot(
-                viewSpan, textSnapshot, GetCaretPointInViewBuffer());
+                viewSpan, textSnapshot, endPoint: stopBeforeCaret ? (int?)GetCaretPointInViewBuffer() : null);
```

Comment:
I haven't verify what was the behavior in VS2013 when trying to wrap something before existing method call, it may not make one disappointed.